packages/dds/shared-object-base/src/serializer.ts

```diff
@@ -66,7 +68,9 @@ export interface IFluidSerializer {
 
 /**
  * Data Store serializer implementation
- * @alpha
+ * @privateRemarks
+ * used by fluidframework/shared-object-base public SummarySerializer
+ * @public
```

Comment:
I don't think this should be public. We have the interface above for more public use (If needed: I don't think it should be public either). Making shared object not public will help with this.